library/core/src/slice/mod.rs

```diff
@@ -2692,8 +2692,9 @@ impl<T> [T] {
     /// This reordering has the additional property that any value at position `i < index` will be
     /// less than or equal to any value at a position `j > index`. Additionally, this reordering is
     /// unstable (i.e. any number of equal elements may end up at position `index`), in-place
-    /// (i.e. does not allocate), and *O*(*n*) worst-case. This function is also/ known as "kth
-    /// element" in other libraries. It returns a triplet of the following from the reordered slice:
+    /// (i.e. does not allocate), and *O*(*n*) on average. In some cases this function performs *O*(*n*^2).
+    /// This function is also/ known as "kth element" in other libraries.
```

Comment:
This `/` is a bit out of place? I have no idea how it got there. No reason not to remove it just because the copy-editing was bad before.

Reply:
Oh, I thought it was a formatting mark

Reply:
Removed.